permissions/plugin-testquality-updater.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "testquality-updater"
+paths:
+- "org/jenkins-ci/plugins/testquality-updater"
```

Comment:
Path does not match effective GA due to https://github.com/jenkinsci/testquality-updater-plugin/blob/f87f4361a45b5b0cd7d2776fe1b78322acaa3f8d/pom.xml#L12

Reply:
Updated groupId, https://github.com/jenkinsci/testquality-updater-plugin/commit/67d01c673a9e7b1067cf64ebf341a3eb353f59e2